lib/fencing/st_client.c

```diff
@@ -2429,6 +2429,44 @@ stonith__event_exit_reason(stonith_event_t *event)
     return ((pcmk__action_result_t *) event->opaque)->exit_reason;
 }
 
+/*!
+ * \internal
+ * \brief Return a human-friendly description of a fencing event
+ *
+ * \param[in] event  Event to describe
+ *
+ * \return Newly allocated string with description of \p event
+ * \note The caller is responsible for freeing the return value.
+ *       This function asserts on memory errors and never returns NULL.
+ * \note This currently is useful only for events of type
+ *       T_STONITH_NOTIFY_FENCE.
```

Comment:
for the case that sbdy still does we might assert or create something short instead of garbage?

Reply:
That makes sense

Reply:
I forgot, that's one of the items on my follow-up list ... currently, xml_to_event() doesn't set event->type or event->message so we don't have it here :-/